projects/storefrontlib/src/lib/occ/docs/cms-component-implementation.md

```diff
@@ -106,8 +106,19 @@ ConfigModule.withConfig({
   cmsComponents: {
     SearchBoxComponent: {
         disableSSR: true
-      ];
     }
   }
 });
 ```
+
+
+# Placeholder components
+
+For Angular (or web-) components that don't need any data from CMS (for example *login*) the CMS component of type `FlexCmsComponent` can be used as a placeholder. This CMS component contains the special attribute `flextype` that will be used in the Spartacus' CMS mapping instead of the original component type.
+
+In the same vein, the `uid` attribute of `JspIncludeComponent` is used in the CMS mapping instead of the original component type.
+
+**Note:** We recommend using `FlexCmsComponent` rather than `JspIncludeComponent`, because:
+
+- `uid` has to be unique, so you can't have two instances of the same `JspIncludeComponent`
+- `flextype` gives you more type safety, because it should come from an enumeration dropdown in the CMS
```

Comment:
This isn't verified with Wei. Can you remove or rephrase, so that we do not make a promise that we can't make true?